internal/workflow.go

```diff
@@ -2401,6 +2401,12 @@ type nexusClient struct {
 //
 // NOTE: Experimental
 func NewNexusClient(endpoint, service string) NexusClient {
+	if endpoint == "" {
+		panic("endpoint must not be empty")
+	}
+	if service == "" {
+		panic("service must not be empty")
+	}
 	return nexusClient{endpoint, service}
```

Comment:
You could argue this validation should be post-interceptor, but I am fine with it as is for now

Reply:
Hmm... yeah, potentially, but I think getting the line number in the stack trace is better for debugging.